content/actions/using-workflows/workflow-syntax-for-github-actions.md

```diff
@@ -413,7 +413,7 @@ Selects an action to run as part of a step in your job. An action is a reusable
 
 We strongly recommend that you include the version of the action you are using by specifying a Git ref, SHA, or Docker tag number. If you don't specify a version, it could break your workflows or cause unexpected behavior when the action owner publishes an update.
```

Comment:
This isn't really your thing, but if someone is going to review this, maybe they should apply this change as well at the same time:

```suggestion
We strongly recommend that you include the version of the action you are using by specifying a Git ref, SHA, or Docker tag. If you don't specify a version, it could break your workflows or cause unexpected behavior when the action owner publishes an update.
```

Afaict there's no such thing as a `docker tag number`, just `docker tags`.

Reply:
I also agree with this; let's see whether the reviewer from GitHub would prefer a separate pull request with this change. Happy to throw an extra commit into this one though.

Reply:
![Uploading 2ACB98DA-19A3-4A00-8B97-AA359B1916B0.png…]()